src/components/Balloon/Balloon.tsx

```diff
@@ -37,6 +37,7 @@ const Base = styled.div`
     return css`
       position: relative;
       display: inline-block;
+      font-size: 12px;
```

Comment:
You must use the theme instead of specifying the value directly.
Please refer to the style of this component.
https://github.com/kufu/smarthr-ui/blob/master/src/components/Flash/Flash.tsx#L141

Although font-size is specified directly in Balloon.stories.tsx, it is acceptable because it has nothing to do with components.

The default font size is specified here.
https://github.com/kufu/smarthr-ui/blob/master/src/themes/createSize.ts#L58-L71